core/controllers/skill_mastery_test.py

```diff
@@ -552,8 +550,7 @@ def test_get_with_valid_topic_ids(self):
                 'comma_separated_topic_ids': ','.join([topic_id_1, topic_id_2])
             })
         degrees_of_mastery_1 = {
-            u'1': python_utils.divide(
-                self.degree_of_mastery_1 + self.degree_of_mastery_2, 2),
+            u'1': ((self.degree_of_mastery_1 + self.degree_of_mastery_2) // 2),
```

Comment:
Why is it `//` here? Shouldn't it be true division in all tests?

Reply:
Uhhh... I missed it. 

Both degree_of_mastery_1 and degree_of_mastery_2 are of float type.